embulk-api/src/main/java/org/embulk/spi/json/JsonArray.java

```diff
@@ -112,6 +112,20 @@ public JsonArray asJsonArray() {
         return this;
     }
 
+    /**
+     * Returns the approximate size of this JSON array in bytes presumed to occupy in {@link org.embulk.spi.Page} as a reference.
+     *
+     * @return the approximate size of this JSON array in bytes presumed to occupy in {@link org.embulk.spi.Page} as a reference
+     */
+    @Override
+    public int presumeReferenceSizeInBytes() {
+        int sum = 4;
```

Comment:
Minor question why 4-byte? (I know it represents the type of JsonArray. But I didn't know Why four-byte)

Reply:
Noted in a comment there in e86e09cb7fa3e00e16bd0ca97641ec681c15c9c0